isthmus/src/test/java/io/substrait/isthmus/CustomFunctionTest.java

```diff
@@ -99,6 +113,51 @@ public RelDataType toCalcite(Type.UserDefined type) {
           null,
           SqlFunctionCategory.USER_DEFINED_FUNCTION);
 
+  static final SqlFunction customScalarAnyFn =
+      new SqlFunction(
+          "custom_scalar_any",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.explicit(SqlTypeName.VARCHAR),
+          null,
+          null,
+          SqlFunctionCategory.USER_DEFINED_FUNCTION);
+
+  static final SqlFunction customScalarListAnyFn =
+      new SqlFunction(
+          "custom_scalar_listany",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.explicit(varcharArrayType),
+          null,
+          null,
+          SqlFunctionCategory.USER_DEFINED_FUNCTION);
+
+  static final SqlFunction customScalarListAnyAndAnyFn =
+      new SqlFunction(
+          "custom_scalar_listany_any",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.explicit(varcharArrayType),
```

Comment:
`custom_scalar_listany_any` is defined as returning a `LIST<any1>`. The return type here doesn't match that.